rfc/0054-sdk3-management-apis.md

```diff
@@ -1494,6 +1510,321 @@ Map where top level keys are dataverse names, and values are a map of dataset na
+
+  * `linkName`: `string` - the link to be removed.
+  * `dataverseName`: `string` - the dataverse in which the link belongs.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+### Returns
+
+Nothing
+
+### Throws
+
+* `DataverseNotFoundException`
+
+* `LinkNotFoundException`
+
+* `InvalidArgumentsException`
+
+### URI
+Data is sent as `"application/x-www-form-urlencoded"`
+
+A payload containing `dataverse` and `name` and URI is:
+
+    DELETE http://localhost:8095/analytics/link
+
+
+## GetLinkFromScope
+
+Gets an existing link from a scope.
+
+### Signature
+
+```
+AnalyticsLink GetLinkFromScope(string linkName, string scopeName [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `linkName`: `string` - the link to be removed.
+  * `scopeName`: `string` - the scope in which the link belongs, in the format `bucket/scope`.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+### Returns
+
+The JSON payload is returned from the server as an array containing one element which must be extracted.
+
+`AnalyticsLink` - the interface type which must be cast to the relevant underlying type.
+
+### Throws
+
+* `AnalyticsScopeNotFoundException`
+
+* `LinkNotFoundException`
+
+* `InvalidArgumentsException`
+
+### URI
+
+`scope` is url encoded to escape any `/` within the name and URI is:
+
+    GET http://localhost:8095/analytics/link/<scope>/<linkname>
+
+
+## GetLinkFromDataverse
+
+Gets an existing link from a dataverse.
+
+### Signature
+
+```
+AnalyticsLink GetLinkFromDataverse(string linkName, string dataverseName [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `linkName`: `string` - the link to be removed.
+  * `dataverseName`: `string` - the dataverse in which the link belongs.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+### Returns
+
+The JSON payload is returned from the server as an array containing one element which must be extracted.
+
+`AnalyticsLink` - the interface type which must be cast to the relevant underlying type.
+
+### Throws
+
+* `DataverseNotFoundException`
+
+* `LinkNotFoundException`
+
+* `InvalidArgumentsException`
+
+### URI
+
+    GET http://localhost:8095/analytics/link?dataverse=<dataverse>&name=<name>
+
+
+## GetAllLinks
+
+Gets all existing links.
+
+### Signature
+
+```
+List<AnalyticsLink> GetAllLinksFromDataverse([options])
+```
+
+### Parameters
+
+* Required:
+
+None
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+  * `Scope` (string) - the name of the scope to restrict links to.
+  * `Dataverse` (string) - the name of the dataverse to restrict links to.
```

Comment:
Are the `scope` and `dataverse` options mutually exclusive?

Reply:
Yes, added a note to specify this.